sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -374,8 +383,8 @@ setQuantifier
 
 relation
     : left=relation
-      ((CROSS | joinType) JOIN right=relation joinCriteria?
-      | NATURAL joinType JOIN right=relation
+      ((CROSS | joinType) JOIN broadcast=broadcastType? right=relation joinCriteria?
```

Comment:
I am not a giant fan or this syntax. I'd prefer `LEFT BROADCAST JOIN`, but I am aware that this is also clunky: 'LEFT RIGHT BROADCAST JOIN`...